src/Entities/Collection.ts

```diff
@@ -98,6 +98,6 @@ export class Collection {
   @Field(type => [CollectionGroup], {
     description: "CollectionGroups of this collection",
   })
-  @Column({ default: [] })
+  @Column(type => CollectionGroup)
```

Comment:
The type specified here says it's a single CollectionGroup, but the type on the data says it's an Array of collection groups - is this a mistake?

Reply:
I was confused about that as well. I double checked the docs and that's the right way to type arrays
https://typeorm.io/#/mongodb/defining-subdocuments-embed-documents

The example they give is 

```typescript
import {Entity, ObjectID, ObjectIdColumn, Column} from "typeorm";

@Entity()
export class User {

    @ObjectIdColumn()
    id: ObjectID;

    @Column()
    firstName: string;

    @Column()
    lastName: string;

    @Column(type => Profile)
    profile: Profile;

    @Column(type => Photo)
    photos: Photo[];

}
```

If you look at `photos`, the type is `Photo` not `Photo[]`